api/app/routers/fba.py

```diff
@@ -165,21 +155,28 @@ async def get_run_datetimes_for_date_and_runtype(run_type: RunType, for_date: da
         return datetimes
 
 
-@router.get('/fire-zone-elevation-info/{run_type}/{for_date}/{run_datetime}/{fire_zone_id}',
-            response_model=FireZoneElevationStatsListResponse)
-async def get_fire_zone_elevation_stats(fire_zone_id: int, run_type: RunType, run_datetime: datetime, for_date: date,
-                                        _=Depends(authentication_required)):
-    """ Return the elevation statistics for each advisory threshold """
+@router.get("/fire-zone-elevation-info/{run_type}/{for_date}/{run_datetime}/{fire_zone_id}", response_model=FireZoneElevationStatsListResponse)
+async def get_fire_zone_elevation_stats(fire_zone_id: int, run_type: RunType, run_datetime: datetime, for_date: date, _=Depends(authentication_required)):
+    """Return the elevation statistics for each advisory threshold"""
     async with get_async_read_session_scope() as session:
         data = []
         rows = await get_zonal_elevation_stats(session, fire_zone_id, run_type, run_datetime, for_date)
         for row in rows:
-            stats = FireZoneElevationStats(
-                minimum=row.minimum,
-                quartile_25=row.quartile_25,
-                median=row.median,
-                quartile_75=row.quartile_75,
-                maximum=row.maximum)
+            stats = FireZoneElevationStats(minimum=row.minimum, quartile_25=row.quartile_25, median=row.median, quartile_75=row.quartile_75, maximum=row.maximum)
             stats_by_threshold = FireZoneElevationStatsByThreshold(threshold=row.threshold, elevation_info=stats)
             data.append(stats_by_threshold)
         return FireZoneElevationStatsListResponse(hfi_elevation_info=data)
+
+
+@router.get("/fire-zone-tpi-stats/{run_type}/{for_date}/{run_datetime}/{fire_zone_id}", response_model=FireZoneTPIStats)
+async def get_fire_zone_tpi_stats(fire_zone_id: int, run_type: RunType, run_datetime: datetime, for_date: date, _=Depends(authentication_required)):
+    """Return the elevation TPI statistics for each advisory threshold"""
+    logger.info("/fba/fire-zone-tpi-stats/")
+    async with get_async_read_session_scope() as session:
+        stats = await get_zonal_tpi_stats(session, fire_zone_id, run_type, run_datetime, for_date)
+        return FireZoneTPIStats(
+            fire_zone_id=fire_zone_id,
+            valley_bottom=stats.valley_bottom * stats.pixel_size_metres,
```

Comment:
Is pixel size stored as area? Or a 1-D length? If the latter, then we might want to square the pixel size.

Reply:
Ultimately it might not matter as we're just displaying percentages in the frontend, not absolute values.

Reply:
Done in: https://github.com/bcgov/wps/pull/3838/commits/a8517bc7a28b3dd86e331c69cb8e70f143b1c56f